core/src/test/java/kafka/test/api/ShareConsumerTest.java

```diff
@@ -1635,7 +1636,7 @@ private <K, V> KafkaShareConsumer<K, V> createShareConsumer(Deserializer<K> keyD
     private void warmup() throws InterruptedException, ExecutionException, TimeoutException {
         createTopic(warmupTp.topic());
         TestUtils.waitForCondition(() ->
-                        !scala.collection.JavaConverters.seqAsJavaList(cluster.brokers().get(0).metadataCache().getAliveBrokerNodes(new ListenerName("EXTERNAL"))).isEmpty(),
+                        !CollectionConverters.asJava(cluster.brokers().get(0).metadataCache().getAliveBrokerNodes(new ListenerName("EXTERNAL"))).isEmpty(),
```

Comment:
We can use `isEmpty` directly even though it is scala code, right?

Reply:
hi  @chia7712 , I have recommitd the code, please have a look😊